network/base/views.py

```diff
@@ -348,6 +357,20 @@ def prediction_windows(request, sat_id, start_date, end_date, station_id=None):
     for station in stations:
         if not station.online:
             continue
+
+        # skip if this station is not capable of receiving the frequency
+        frequency_supported = False
+        for gs_antenna in station.antenna.all():
+            try:
+                if (int(gs_antenna.frequency) <=
```

Comment:
downlink is coming from downlink_low and is a `PositiveIntegerField` so yes we need the int() here